src/parser/priest/discipline/modules/azeritetraits/DeathThroes.js

```diff
@@ -54,66 +52,74 @@ class DeathThroes extends Analyzer {
       return;
     }
 
-    const { damage } = deathThroesStats(this.selectedCombatant.traitsBySpellId[SPELLS.DEATH_THROES.id]);
-    // Reduce damage based on the disc priest damage nerf.
+    const { damage } = deathThroesStats(
+      this.selectedCombatant.traitsBySpellId[SPELLS.DEATH_THROES.id],
+    );
+
     this.damageValue = damage * (1 - DISC_PRIEST_DAMAGE_REDUCTION);
+    this.addEventListener(
+      this.atonementAnalyzer.atonementEventFilter.by(SELECTED_PLAYER),
+      this.atonementHandler,
+    );
+    this.addEventListener(
+      this.atonementAnalyzer.atonementDamageSourceFilter,
+      this.handleDeathThroesDamage,
+    );
   }
 
-  on_byPlayer_damage(event) {
-    const spellId = event.ability.guid;
-    if (spellId !== SPELLS.SHADOW_WORD_PAIN.id && spellId !== SPELLS.PURGE_THE_WICKED_BUFF.id && !event.ability.tick) {
-      return;
-    }
-    this.damageDone += this.damageValue;
+  /**
+   * Processing this here ensures that the damage event is only considered once,
+   * so we don't get super inflated values for damage contribution
+   *
+   * @param {Object} damageEvent A damage event
+   */
+  handleDeathThroesDamage(damageEvent) {
```

Comment:
This is an example of how to reverse engineer the correct value for a static buff trait:
* Calculate the theoretical base for the item
* Look at the damage event for the spell it buffs, check for the total damage by adding raw and absorbed damage.
* Look at the `unmitigatedAmount`, this works for both damage buffs and reductions. For example, a player hitting for a 100% damage buff will have '200' in their `amount + absorb`, but the `unmitigatedAmount` will be '100'.
* Use this to calculate a ratio to multiply the theoretical trait value by.